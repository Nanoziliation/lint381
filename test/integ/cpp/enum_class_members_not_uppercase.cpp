enum class Foo {
    FOO,
    bar,
    BAZ /*QUX*/
    /*GRAULT*/
};
