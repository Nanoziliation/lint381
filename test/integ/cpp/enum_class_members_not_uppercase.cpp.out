enum_class_members_not_uppercase.cpp:2:5: error: Enum class member 'FOO' should not be uppercase
    FOO,
    ^^^
enum_class_members_not_uppercase.cpp:4:5: error: Enum class member 'BAZ' should not be uppercase
    BAZ /*QUX*/
    ^^^
